web/src/main/java/se/inera/intyg/webcert/web/service/facade/util/UtkastToCertificateConverterImpl.java

```diff
@@ -124,6 +128,13 @@ private Certificate convertToCertificate(Utkast certificate) {
                 certificateToReturn.getMetadata().getTypeVersion())
         );
 
+        certificateToReturn.getMetadata().setRecipient(
+            certificateRecipientConverter.get(
```

Comment:
Use this recipient to replace the code that used for "sentTo" (further up in this class) to find the recipient name. By using the new CertificateRecipientConverter where we fetches this from IT we can get rid of the RecipientConverter (with the hardcoded recipients).